index.js

```diff
@@ -122,7 +120,7 @@ function ExtractTextPlugin(options) {
 	if(isString(options)) {
 		options = { filename: options };
 	} else {
-		schemaTester(pluginSchema, options);
+		validateOptions('schema/plugin.json', options, 'Extract Text Plugin');
```

Comment:
Just to double check `=> ./ <= schema...` ?  (validateOptions use `require()` 'internally')